tests/openjpeg/syntensity_lobby_s.raw

```diff
@@ -1,2 +1,2 @@
-&:��l:{�>Mw9-)Hg?R,(1EJEb���GA>VJE@��?(-/.m��*m�Y?�U#"DmB9/&5ukFu�|[KEMZV:d��',,-04][9@SA/I8)%9bHR*%6iVHf|xeV\^^�>��S%$#&-)��EÿK��'.CbIQ)(.Cq��ڣ����yxu��("(,!%���M��{��86?A>@  4f�������Nq�k,-!$0:ȿQx�q_y1@y97/!+G�㵄����J��:%$.0@.2+'% " *<j6)" #-D{�x��j���e:�s*$-/3=+=RdtuoeQ@=65\:($$)0>�{~z{^_X^D$%'(()**_u���������d_U?3! !$&)��S8$(/D2+(!#&(((Z[Xdv������ZH?A=31+,+0��U>"%7?/-*'&'&""KEGLS\dmwuy|]JHF8<3/*1R���{K:4/&*,*()&$$GJRX^jnv���KsZJ94  ,-Q���Ί=$$%&()'"%*T\egqwz�����u��J.AN^j^=8T����="$%%#""&.empv��������ɩ�~���fGXaJ;-Lɓ5   "&��������������⾏�ct[\R<984@G%## �������������ʹ�q@E>6/*47K<)"+## ����������̿�����])$29.!#92#$-  $$ �����ʿ��Ŀ�������s�0 4=,*+.! '$" ����������������nJc9,*8.$,#$&3,+& ����������������wU+3X8&.<0)!0'%10*"��������������vzsk^%I[%'+#)aG.#&'$ ������������uvphusy3/P@??%8@+DF6+&%#����������xutjry~}h7#8+#.59920(*%�������}tv~vrw{~lmiMIFIDE+)3B926/+###����uxrqyw{vlhciklmkcZXC*(=?1)0'#"��uiqvnqv|{{ukhdehkmme_[_OD7+/,+#!%rgjnlnnxt|�tnnmvplqria_a_MIC:&%'e�("oqoklutu|yxswtyxtgmpf_gb[UJLD3A2Csqsqmqku|qu�yvlpcptkggcPXTPQN�=
uvujd`bs}q}�xumfljmoedgY]YSMPTxO4%'B��A��G]�C--SzFb1,7MSOg�ŽOC<TG?E��E.1./���.��hI�e%%O}L@3):�vIw�}WD=K\[6tȘ*0.,.8ki@H`L5WC+'BqR[/'6hR>\wwh[a^^�9��['&%&-+��K��S��+2QpP\..)5Kq�ɧ���mqi��*$)-#%���O����:@HHDE%$'Jr��~������Gs�l,,"%0:��T|�wb}3BzA;4  #)]�wq�h��F��9($-/B!-1,)'"!%#!,@m;/%!$#%GoP{G=v�v\7�t+%,.2?-=RduvqgUJL??e@,'$)$#eHD<=020B:$'(**+,+c{�������ͮxpcF:'""% �{31-+(!$'))(cdbn�������cPGHF7725'��!.'-*'&&&""RLMU[dks|xxoDEFC5:52%"$?FJ2'$*,*()&$$LOTY]egksoqv&WG=., ESWW=%$%&()'#%*SV]]ehjq����Ypr;+4=HPH-'%4d�]-!$%%$"#'.[`bep~��������s^dqmS:EJ4&"(gO2  !!"" &vyx����������¯�sqOZIF=.((+/2"!#%! ���������������}Y694.(#*+2))$3(&  ����������������ugE%(-3-&$-)%*/%' !�������������~}{shU[+$06) "-00!&!$!  �����������}twpicS3=*..6+%*',#&!%$  "������{����uspjg]9 1.6=0.&(! +*% �����{��}�yok^`[TH0<$1+'*F2'"$$" %#������zswzq^_ZU_\b&#@3-7#/3#8;-&$%%#%'�����~qqkob]^V]cggT'-)%-0/)+')&"#$$}��}{xkf^aha^cehYZV@=;>:<)$+84.0-+%%%$$#{y{iac^]dicfbZWSXZ[[ZSKL;-*88.,1* !"$$&�xbW_c[^bgghcZXUTWZ\[VQLQC>9./,-("#!""#(`VY][\\fbilc\][c_[aaZSQSRB@@8(*(!OW#B2"_`]Z[dbcifecechecX^`YRZTOJAC;.9"%#)+!2)"baba\_[dh`domhf\aV`d\YYUGMIGHB`6&%! "efd[UQTak`lugd]V]\``YWZNSPJDGKYA!%!'?1$ $I��G��Qu�P*/Y�Mv2.;OTTe���SC7XH?P��H15+.�ѽ/��~W�{%%V�QF5)=��Jv��XA;J`c6��+30*,9pqBMnU:hN,%HwVc/'7rU9Vmne`e^`�7��_'%$$)&��H��Q��(/QqP]0.+,8d������_dL��(#&)!"���I��x��5:=A?<%##;1xvr������Be�Y)(",5ȿOs�pWp.;j>7-!#"2elrE��C��1'"*+9*-'%$ %#!,0B53&"$%",=T%ks]Q5tc($)*/:)9L_oqldSII=4,-'$(3$#/6$$&)()**ay���������aVJ*()!"$xE	
```

Comment:
This change worries me a bit.    It looks like it might break a test?

Reply:
Ah.. it looks like it does cause the openjpeg tests to fail.

Reply:
Fixed